scan/src/pages/ValidatorIndexPage.re

```diff
@@ -132,14 +173,39 @@ module Uptime = {
   };
 };
 
-[@react.component]
-let make = (~address, ~hashtag: Route.validator_tab_t) => {
-  let validatorSub = ValidatorSub.get(address);
-  let bondedTokenCountSub = ValidatorSub.getTotalBondedAmount();
-
-  let allSub = Sub.all2(validatorSub, bondedTokenCountSub);
+module UptimeMobile = {
+  [@react.component]
+  let make = (~consensusAddress) => {
+    let uptimeSub = ValidatorSub.getUptime(consensusAddress);
+    switch (uptimeSub) {
+    | Data(uptimeOpt) =>
+      switch (uptimeOpt) {
+      | Some(uptime) =>
+        kvRowMobile(
+          "UPTIME",
+          VReactElement(
+            <div className=Styles.headerMobile>
+              <Text value={uptime |> Format.fPercent(~digits=2)} weight=Text.Bold code=true />
+            </div>,
+          ),
+        )
+      | None => kvRowMobile("UPTIME", VText("N/A"))
+      }
+    | _ =>
+      kvRowMobile(
+        "UPTIME",
+        VReactElement(
+          <div className=Styles.headerMobile> <LoadingCensorBar width=100 height=16 /> </div>,
+        ),
+      )
+    };
+  };
+};
```

Comment:
Do need to create a new one for mobile. Let's use `Uptime` module. In the `Uptime` module, I just see the duplicated title and description like "Percentage of the blocks that the validator is active for out of the last 250".